whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -17508,12 +17548,13 @@
       ]
     },
     "773": {
-      "include": ["referenceToWork"],
-      "aboutEntity": "?work",
-      "aboutAlias": "_:work",
+      "include": ["referenceToInstanceSansDisplayText"],
+      "aboutEntity": "?thing",
```

Comment:
It could be prudent to add a note like: `"NOTE": "We deviate from https://www.loc.gov/bibframe/mtbf/ as we do not link from the Work with hasInstance.",` to all fields where we do so. (And update the note in 776 which just says "BF2" (which is incorrect as BF2 itself allows this).)

Reply:
Fix it 👍